app/src/main/java/org/easyrecipe/features/search/SearchViewModel.kt

```diff
@@ -64,6 +55,7 @@ class SearchViewModel @Inject constructor(
             }
         ).onSuccess { result ->
             recipeList.value = result.recipes
+            mainViewModel.searchResultList.value = recipeList.value
```

Comment:
An alternative to this case would be to create an event using a `LiveData<List<Recipe>>` inside `SearchViewModel` and observe it in `SearchFragment`. The observable method should assign the value to the corresponding property from `MainViewModel`. Do not forget that you should use the backing field pattern by creating a private property of type `MutableLiveData<List<Recipe>>